docs/getting-started/agentless-integrations.asciidoc

```diff
@@ -3,6 +3,31 @@
 
 Agentless integrations provide a means to ingest data while avoiding the orchestration, management, and maintenance needs associated with standard ingest infrastructure. Using agentless integrations makes manual agent deployment unnecessary, allowing you to focus on your data instead of the agent that collects it. 
 
-We currently support one agentless integration: cloud security posture management (CSPM). Using this integration's agentless deployment option, you can enable Elastic's CSPM capabilities just by providing the necessary credentials. Agentless CSPM deployments support AWS, Azure, and GCP accounts.
+We support the following integrations:
 
-To learn more about agentless CSPM deployments, refer to the getting started guides for CSPM on <<cspm-get-started, AWS>>,  <<cspm-get-started-azure, Azure>>, or <<cspm-get-started-gcp, GCP>>. 
\ No newline at end of file
+[discrete]
+== Generally available (GA) agentless integrations
+Cloud security posture management (CSPM). Using this integration's agentless deployment option, you can enable Elastic's CSPM capabilities just by providing the necessary credentials. Agentless CSPM deployments support AWS, Azure, and GCP accounts.
+
+To learn more about agentless CSPM deployments, refer to the getting started guides for CSPM on <<cspm-get-started, AWS>>,  <<cspm-get-started-azure, Azure>>, or <<cspm-get-started-gcp, GCP>>. 
+
+## Beta agentless integrations
```

Comment:
Is this supposed to be a L2 header? In the [preview](https://security-docs_bk_6712.docs-preview.app.elstc.co/guide/en/security/8.x/agentless-integrations.html#_beta_agentless_integrations), it's appearing as a sub-section with in the GA agentless integration's section.


```suggestion
== Beta agentless integrations
```